README.md

```diff
@@ -66,6 +66,8 @@ these are defined in the repo [aws-account-wide-terraform][]
 $ export DEPLOY_ENV=environment-name
 ```
 
+It is important that you do not use the same `DEPLOY_ENV` for both build and deployer concourse environments, this is to avoid conflicts in resource allocation.
```

Comment:
I'm wondering if replacing `important` with `recommended` would be a better fit here.